### PR TITLE
browser_style:true removed from MV3

### PR DESCRIPTION
**Pull request by rebloor**
### Description

Add a release note to indicate that 'browser_style:true' removed from MV3 in release 118.

### Related issues and pull requests

Provide documentation for [Bug 1830711](https://bugzilla.mozilla.org/show_bug.cgi?id=1830711) Turn off support for browser_style:true in MV3
Related browser compatibility data changes in [#20480](https://github.com/mdn/browser-compat-data/pull/20480)

### CODE REVIEW

**Comment by rebloor** [comment]
@Rob--W are you happy to change the references on the manifest key files from "support will be removed in Firefox 118." to "support removed in Firefox 118." or do that on 26 September?

**Review by Rob--W — APPROVED** [review]
This looks good. There are still references in the future tense like "will be removed in Firefox 118", added in https://github.com/mdn/content/pull/26642. Do you want to rephrase these before merging?

**Comment by rebloor** [comment]
@Rob--W thanks for answering my question, although I think you missed it.

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/action/index.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ The `action` key is an object that may have any of these properties, all optiona
         <p>Optional, defaulting to <code>false</code>.</p>
         <div class="notecard warning">
           <p>
-            Do not set <code>browser_style</code> to true: it is deprecated in Manifest V3, and support will be removed in Firefox 118. See <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles#manifest_v3_migration">Manifest V3 migration for <code>browser_style</code></a>.
+            Do not set <code>browser_style</code> to true: its not support in Manifest V3 from Firefox 118. See <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles#manifest_v3_migration">Manifest V3 migration for <code>browser_style</code></a>.
```
Comment (Rob--W) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/action/index.md:87]:
```suggestion
            Do not set <code>browser_style</code> to true: its support in Manifest V3 was removed in Firefox 118. See <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles#manifest_v3_migration">Manifest V3 migration for <code>browser_style</code></a>.
```

"its not support" doesn't look like proper English. Here is a suggestion to fix it.

File: files/en-us/mozilla/firefox/releases/118/index.md [hunk 2]
```diff
@@ -58,6 +58,8 @@ This article provides information about the changes in Firefox 118 that affect d
 
 ### Removals
 
+- Support for [`browser_style`](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles) in the manifest keys [`action`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action), [`options_ui`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui), [`page_action`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action), and [`sidebar_action`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/sidebar_action) for Manifest V3 extensions has been removed ([Firefox bug 1830711](https://bugzil.la/1830711)). See [Manifest v3 migration](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles#manifest_v3_migration) for information about transitioning from `browser_style` in Manifest V3 extensions.
```
Comment (Rob--W) [inline files/en-us/mozilla/firefox/releases/118/index.md:61]:
```suggestion
- Support for [`browser_style`](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles) in the manifest keys [`action`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action), [`options_ui`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui), [`page_action`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action), and [`sidebar_action`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/sidebar_action) for Manifest V3 extensions has been removed ([Firefox bug 1830711](https://bugzil.la/1830711)). See [Browser Styles' Manifest v3 migration](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles#manifest_v3_migration) for information about transitioning from `browser_style` in Manifest V3 extensions.
```

Optional suggestion: Prepend "Browser Styles" to link to emphasize that it is not a generic article but specific to Browser Styles.

**Review by Rob--W — COMMENTED** [review]
Feel free to merge after fixing the English issue.

**Pull request merged**